_docs/style/focus-states.md

```diff
@@ -3,424 +3,235 @@ title: Focus States
-
-{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
-{% figure [class:"image bg-light border"] %}
-<img src="{{site.url}}/assets/images/style/focus-states/focus-states-reversed-focus-color.png" width="249"/>
-{% endfigure %}
-{% endcolumn %}
-</div>
-
-
 </section>
 
 <section class="static-section" markdown="1">
 
 <div class="hxRow" markdown="1">
+
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Irregular shapes
-Apply the standard glow and a 2px corner radius to elements that do not have a visible box shape. 
+### Composition
 
-Examples of this are tertiary buttons, hyperlinks, and button icons such as the cog or close button on alerts, toasts, and modals.
+Focus consists of the following elements:
 
-{% endcolumn %}
+- **Glow (required)**: The glow is the hazy ring that appears around the focused element.
+- **Border (optional)**: Apply a border only to elements that already have a border.
 
-{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
-{% figure [class:"image bg-light border"] %}
-<img src="{{site.url}}/assets/images/style/focus-states/focus-states-irregular-shapes.png" width="283"/>
-{% endfigure %}
 {% endcolumn %}
 
-</div>
-</section>
-
-<section class="static-section" markdown="1">
-
-## Focusable elements
-
-<div class="hxRow" markdown="1">
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
-
-### Hyperlinks and text
-
-Hyperlinks and focusable text are styled using the standard glow, with a 2px corner radius. The height of the glow is determined by the line height of the text. 
-
-#### Multi-line text focus
-If the text breaks across multiple lines, indicate continuation by using 0px corners at the end of the preceding line and the start of the subsequent line.
-
-#### Elements: 
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-- [Links]({{site.baseurl}}/style/typography.html#link)
-- [Breadcrumbs]({{site.baseurl}}/components/breadcrumbs.html)
+{% figure [caption:"Focus state composition"] [class:"image bg-light border"] %}
 
-{% endcolumn %}
+<img src="{{site.url}}/assets/images/style/focus-states/focus-composition.png" width="213"/>
 
-{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
-{% figure [class:"image bg-light border"] %}
-<img src="{{site.url}}/assets/images/style/focus-states/focus-states-hyperlinks-and-text.png" width="455"/>
 {% endfigure %}
+
 {% endcolumn %}
 
 </div>
+
 </section>
 
 <section class="static-section" markdown="1">
 
 <div class="hxRow" markdown="1">
+
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Buttons
+### Element with light fill or border
 
-In addition to primary, secondary and tertiary buttons, the following are a few elements that are also treated as buttons: 
+For elements without a border, if the element's fill color is 500 or lighter, use #0E94A6 (Cyan 700) for the glow color.
```

Comment:
I believe we've been putting hex codes and HTML colors in code fences elsewhere. Are we changing that? I don't want to point it out everywhere if that's not the case.

```suggestion
For elements without a border, if the element's fill color is 500 or lighter, use `#0E94A6` (`cyan 700`) for the glow color.
```

Reply:
Ah yes, ok, made that change throughout. Thanks!